WordPress/src/main/java/org/wordpress/android/ui/pages/PagesActivity.kt

```diff
@@ -26,4 +33,32 @@ class PagesActivity : AppCompatActivity() {
         }
         return super.onOptionsItemSelected(item)
     }
+
+    override fun onPositiveClicked(instanceTag: String) {
+        passDeleteConfirmation(instanceTag.toLong())
+    }
+
+    override fun onNegativeClicked(instanceTag: String) {
+    }
+
+    private fun passDeleteConfirmation(remoteId: Long) {
+        val fragment = supportFragmentManager.findFragmentById(id.fragment_container)
+        if (fragment is PagesFragment) {
+            fragment.onPageDeleteConfirmed(remoteId)
+        }
+    }
+
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        if (requestCode == RequestCodes.EDIT_POST && resultCode == Activity.RESULT_OK && data != null) {
```

Comment:
maybe just pass the `onActivityResult` down to the fragment so it can decide what to do with it? There would be less logic in the activity

Reply:
Good point.